content/articles/automating-windows-applications-withpython/index.md

```diff
@@ -0,0 +1,146 @@
+- [Automating notepad](#automating-notepad)
+- [Automating Excel with Pywinauto](#automating-excel-with-pywinauto)
+- [Python automation tools](#python-automation-tools)
+- [Conclusion](#conclusion)
+
+
+### Prerequisites
+To follow along, the reader must have a good knowledge of [Python](https://www.python.org).
+
+### Areas where automation can be used
+Let's take a look at some of the most common automation scenarios before going on:
+ - **Service from an automated support center:** - Various firms or organizations give assistance centers where customers may ask problems by email, SMS,, or other means and receive a fast response. 
+ - **Auto-filling a form:** - Automated scripts can easily handle the tedious work of filling out online forms and restoring forgotten credentials, among other things. 
+ - **Quick Climate reports:** - 
+You may also check the weather simply and quickly, but it is more efficient if it is delivered to you automatically. As a result, this is a case of automation in which you are automatically notified of daily weather updates.
+ - **Automatic mail triggers daily:**- This is useful if you have to send an attachment every day. This can be readily managed with an automatic script that runs every day. 
+### Installation of Pywinauto
+Command to install Pwinauto:
+```bash
+pip install pywinauto 
+```
+
+To validate yout installation run this code snippet below:
+```python
+from pywinauto.application import Application
+app = Application(backend="uia").start("notepad.exe") # open notepad app
+app.UntitledNotepad.type_keys("%FX")
+```
+This should start up your notepad application on the windows interface.
+After this installation and validation, we will jump to notepad automation which is the simple technique, followed by excel automation where we will read data from a spreadsheet and write data to the excel cells.
+
+### Automating notepad
+This example will give us an insight into how pywinauto functions by looking at functionalities like starting an application, maximizing and minimizing the application, and writing text in a notepad application. To begin, let us first import the relevant fields and launch notepad using the start method of the application imported.
+
+```python
+from pywinauto import application
+ap = application.Application ()
+ap.start('Notepad.exe')
+```
+When the above code is executed notepad application is launched. Let's add some text on the opened notepad application. The message we intend to display is passed as a string in the text area inside the `type_key()s` method.
+
+```python
+text_message = "Hello, this is a sample text for python automation"
+dlg.typekeys(text_message, with_spaces = True)
+```
+We can use the `is_maximize ()` method to maximize the notepad window as in most cases when notepad is started its window is not in full screen always. 
+```python
+#maximize the notepad window 
+ if dlg.is_maximized () == False:
+    dlg.maximize()
+```
```

Comment:
I remember that in your previous PR I had asked you to cover few other instances of the Notepad usecases - The article is too short to be published (contentwise or value wise)

Reply:
You have just covered maximizing screen of notepad - it can be any application.

It's not notepad specific